src/core/Input/InputField/InputField.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import IconButton from '@material-ui/core/IconButton';
```

Comment:
Can we import all the similar related components from @material-ui/core and  @material-ui/icons at one place respectively.

Import { a , b , c  } from '@material-ui/core similarly.